app/models/game.rb

```diff
@@ -5,16 +5,6 @@ class Game < ActiveRecord::Base
 
 	after_create :populate_board!
 
-
-	def index
-		@list_all_games=Game.all
```

Comment:
Yes, for some reason I was putting things in the games model that should
have been in the games controller.

I'll get to work on the games.each code rewrite. In the meantime, I was
thinking to go ahead and merge into master so that others can start to
branch and work on other tasks.

Is this good practice? Or should we only merge branches that are perfect?

On Sun, Oct 4, 2015 at 2:13 PM, Anna <notifications@github.com> wrote:

> In app/models/game.rb
> <https://github.com/firehoseCheckmates/ourGame/pull/26#discussion_r41100439>
> :
>
> > @@ -5,16 +5,6 @@ class Game < ActiveRecord::Base
> >
> >  	after_create :populate_board!
> >
> > -
> > -	def index
> > -		@list_all_games=Game.all
>
> David and I were reading the code yesterday so we moved it. [image:
> :smiley_cat:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/firehoseCheckmates/ourGame/pull/26/files#r41100439>.
>